src/Hedgehog/Gen.fs

```diff
@@ -47,80 +38,58 @@ module Gen =
 
           Tree.bind (run seed1 m) (run seed2 << k)
 
-    [<CompiledName("Bind")>]
     let bind (m0 : Gen<'a>) (k0 : 'a -> Gen<'b>) : Gen<'b> =
         bindRandom (toRandom m0) (toRandom << k0) |> ofRandom
 
-    [<CompiledName("Apply")>]
+    let bindFlipped (k0 : 'a -> Gen<'b>) (m0 : Gen<'a>) : Gen<'b> =
```

Comment:
Where would we need this?

Reply:
Good catch!

I think that's a left over from an earlier experiment, I'll remove it.